### PR TITLE
add format-triplets hook

### PR DESCRIPTION
**Pull request by davidstrouk**
add format-triplets hook in order to forbid specific triplets referenced multiple times that break the UI

### CODE REVIEW

**Review by nitzo — COMMENTED** [review]
@davidstrouk 
- Add a test ? So we can understand the issue ? Not sure I follow it.
- These relations seem very specific and don't belong in a general open-source liniting library. Let's consider passing them as parameters to the pre-commit hook ?
- https://pre-commit.com/#hooks-args

**Comment by davidstrouk** [comment]
> @davidstrouk
> 
> * Add a test ? So we can understand the issue ? Not sure I follow it.
> * These relations seem very specific and don't belong in a general open-source liniting library. Let's consider passing them as parameters to the pre-commit hook ?
> * https://pre-commit.com/#hooks-args

@nitzo good idea, i'll do that

File: lint/format_triplets.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import csv
+from typing import Optional, Sequence
+
+import gamla
+
+# def _duplicate_object(rows):
+#     def _duplicate_object(object_key):
+#         return gamla.pipe(rows, gamla.filter(gamla.))
+#
+#     return _duplicate_object
```
Comment (molaxx) [inline lint/format_triplets.py:11]: delete please

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -12,9 +12,16 @@
...
   description: Format CSV files (remove empty lines + sort).
-  entry: csv-format
+  entry: format-csv
   language: python
-  files: \.(csv)$
+  files: triplets\.csv$
+
+- id: format-triplets
+  name: format triplets files
+  description: Format CSV files (remove empty lines + sort).
```
Comment (molaxx) [inline .pre-commit-hooks.yaml:24]: make description and name informative

**Comment by molaxx** [comment]
LGTM

**Comment by molaxx** [comment]
LGTM

**Pull request merged**